core/src/main/java/org/infinispan/statetransfer/OutboundTransferTask.java

```diff
@@ -85,7 +86,7 @@
 
    private InternalEntryFactory entryFactory;
 
-   public OutboundTransferTask(Address destination, Set<Integer> segments, int stateTransferChunkSize,
+   public OutboundTransferTask(Address destination, Set<Integer> segments, int chunkSize,
```

Comment:
It's going into a `CopyOnWriteArraySet` anyway, so it doesn't really matter where the boxing happens.